addons/clones/addon.json

```diff
@@ -12,6 +12,10 @@
     {
```

Comment:
we probably need to update the description

Reply:
"description": "Adds a counter above the stage in the editor which shows the total amount of clones. There is also an option to add a counter to individual sprites.",

Reply:
Doneeeeeeeeee